src/components/common/Directors.vue

```diff
@@ -831,98 +786,105 @@ export default class Directors extends Mixins(CommonMixin, DateMixin, DirectorMi
 
   /**
    * Local helper to provide a complete address object including missing/blank fields.
-   * @returns An address object.
+   * @returns a complete address object
    */
-  private formatAddress (address): AddressIF {
+  private formatAddress (address: any): AddressIF {
     return {
-      'addressCity': address.addressCity || '',
-      'addressCountry': address.addressCountry || '',
-      'addressRegion': address.addressRegion || '',
-      'deliveryInstructions': address.deliveryInstructions || '',
-      'postalCode': address.postalCode || '',
-      'streetAddress': address.streetAddress || '',
-      'streetAddressAdditional': address.streetAddressAdditional || ''
+      addressCity: address.addressCity || '',
+      addressCountry: address.addressCountry || '',
+      addressRegion: address.addressRegion || '',
+      deliveryInstructions: address.deliveryInstructions || '',
+      postalCode: address.postalCode || '',
+      streetAddress: address.streetAddress || '',
+      streetAddressAdditional: address.streetAddressAdditional || ''
     }
   }
 
-  /**
-   * Function called externally to set the draft date.
-   * FUTURE: change this to a prop
-   * @param date The draft date to set.
-   */
-  public setDraftDate (date): void {
```

Comment:
This was originally needed for some logic in this file. But that logic belongs in the parent, so I moved it.